### PR TITLE
#11 Travis ci integration

### PR DESCRIPTION
**Pull request by MichaelDysart**

### CODE REVIEW

File: src/test/java/webApp/webControllerTest.java [hunk 1]
```diff
@@ -18,6 +18,6 @@
 
     @Test
     public void testRestApplication() throws Exception {
-        ;
+
```
Comment (Fahid-Mannan) [inline src/test/java/webApp/webControllerTest.java:21]: This was here so IntelliJ allows you to insert a breakpoint in this line

File: src/main/java/webApp/webController.java [hunk 2]
```diff
@@ -20,7 +25,7 @@ public String index(Model model){
     }
 
     @PostMapping("/createSurvey")
-    public String createSurvey(@RequestParam("surveyName") String surveyName, @RequestBody ArrayList<Questions> questions) {
+    public String createSurvey(@RequestParam("surveyName") String surveyName, @RequestBody Collection<Question> questions) {
```
Comment (TaylorAbraham) [inline src/main/java/webApp/webController.java:28]: Nice work with abstractions here

File: src/test/java/webApp/webControllerTest.java (hunk 1 above)
Comment (TaylorAbraham) [inline src/test/java/webApp/webControllerTest.java:21]: This is the best test ever, I miss that semicolon already

File: src/main/java/webApp/ServingWebContentApplication.java [hunk 3]
```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 
-@SpringBootApplication
+@SpringBootApplication(scanBasePackages={"repository"})
+@EntityScan(basePackages = {"repository"})
```
Comment (Fahid-Mannan) [inline src/main/java/webApp/ServingWebContentApplication.java:8]: maybe have the word repository as a constant as it is used very often as a plain string

**Review by Fahid-Mannan — DISMISSED** [review]

**Review by TaylorAbraham — APPROVED** [review]
Overall, excellent work Michael. You are a valuable contributor to the team. Looking forward to your next pull request in our SYSC 4806 SurveyMonkey GitHub repository project for Carleton University.

**Pull request merged**